CHANGES.rst

```diff
@@ -36,6 +36,10 @@ Unreleased
 * The file for ModActions (praw/models/modaction.py) has been moved to
   praw/models/mod_action.py and the previous has been Deprecated.
 
+**Removed**
```

Comment:
This actually doesn't need to be documented since `_methods` are considered private.

Reply:
@bboe it’s due to the fact that any positional params  could be used, so I decided to put a warning that all parameters would be removed.

Reply:
@bboe this isn't a private method. It's a public method that had an unused parameter `_unmute`.

Reply:
You're right that positional params could be used, but again, they shouldn't in this case because that particular positional param is undocumented. Undocumented private params don't require a changelog entry, nor a major version bump to change/remove.

Reply:
So should I delete the changelog entry?

Reply:
Yes, please.